Product/Production/Services/DocumentQueryCore/src/main/java/gov/hhs/fha/nhinc/docquery/model/builder/impl/DocumentMetadataResultModelBuilderImpl.java

```diff
@@ -74,12 +77,54 @@ public void build() {
         String creationDate = helper.getSingleSlotValue(ResponseSlotName.creationTime, extrinsicObject);
         String serviceStartTime = helper.getSingleSlotValue(ResponseSlotName.serviceStartTime, extrinsicObject);
         String serviceStopTime = helper.getSingleSlotValue(ResponseSlotName.serviceStopTime, extrinsicObject);
+        //language code
```

Comment:
I'm always a fan of extra documentation, but commenting what's going to be set right before it is (e.g., "// language code" right above ".setLanguageCode") clutters the code.  We should remove those lines to make the code more readable, and possibly comment at a high-level what the builder is doing.